packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js

```diff
@@ -1,9 +1,9 @@
 import harden from '@agoric/harden';
-import { allComparable } from '@agoric/ertp/util/sameStructure';
+import { allComparable } from '@agoric/same-structure';
+import makeStore from '@agoric/store';
```

Comment:
doublecheck, but in my tree `makeStore` is not referenced elsewhere in this file, so maybe we can just delete this line

Reply:
Yup, good catch